_episodes/01-introduction.md

```diff
@@ -6,18 +6,21 @@ questions:
 - "What are containers, and why might they be useful to me?"
 objectives:
 - "Show how software depending on other software leads to configuration management problems."
-- "Identify the problems that software installation problems can pose for research."
+- "Identify the problems that software installation can pose for research."
 - "Give two examples of how containers can solve software configuration problems." 
 keypoints:
 - "Almost all software depends on other software components to function, but these components have independent evolutionary paths."
-- "Projects involving many software components can rapidly run into a combinatoric explosion in the number of software version configurations available, yet only a subset of possible configurations actually works as desired."
-- "Containers collect software components together and can help avoid software dependency problems."
-- "Virtualisation is an old technology that container technology makes more practical."
+- "Small environments that contain only the software that's absolutely needed for a given task are easier to replicate and maintain."
+- "Critical systems that cannot be upgraded, due to cost, difficulty, etc. need to be reproduced on newer systems in a maintainable and self-documented way."
+- "Virtualization was created to allow multiple environments to run on a single computer."
+- "Virtual environments allow pipelines to leverage tools from conflicting operating systems (or even conflicting versions!) simply and reliably without additional cost or hardware." 
+- "Containerization takes virtualization one step further by making even smaller environments by avoiding all but the minimum software required."
```

Comment:
I understand what you're saying, but I think the phrase "the minimum software required" avoids questions like "what's the virtualization layer?", "how does a container optimize access to resources better than a VM?", and other details the student's don't really need to know and just adds to the cognitive load.

I agree it would be good to point out that even a "simple" executable relies on software contained in the OS and cannot be run in a "void" container, but I balk at going into the nitty gritty differences between VMs and containers.